### DISCUSSION TITLE
How create a record query from infinity plug in

### DISCUSSION THREAD

**Discussion by ManuRom30** [grafana/grafana-infinity-datasource#646] [Q&A]
Hi, I need to get a record query or alert rule from a JSON response but I've couldn't, I reach to prepare the query and the panel show a numeric result but when I try to create a record query it show up this: "the query didn't return any data, and ask to check the range time" but it es the same of panel. Could they help me please ?    

Best regards Manuel

**Comment by yesoreyeram** [comment]
Can you share your json model and more details on what you are setting up as recorded query? Any screenshots will be helpful.
